### PR TITLE
feat(discord): add guild selection where bot should work

### PR DESCRIPTION
**Pull request by Banonevar**
###### CHECKLIST

- [ ] tests are changed or added
- [ ] documentation is changed or added
- [x] locales are changed or added
- [ ] db relationship (tools/database) are changed or added
- [ ] commit message follows [commit guidelines](https://github.com/sogehige/sogeBot/blob/master/CONTRIBUTING.md#commit-guidelines)

### CODE REVIEW

File: src/bot/integrations/discord.ts [hunk 1]
```diff
@@ -61,27 +61,37 @@ class Discord extends Integration {
...
   @ui({
     type: 'discord-channel',
+    if: () => self.guild.length === 0 || self.guild.length === 0,
   })
   sendGeneralAnnounceToChannel = '';
 
   @settings('mapping')
   @ui({
     type: 'discord-mapping',
+    if: () => self.guild.length === 0 || self.guild.length === 0,
```
Comment (sogehige) [inline src/bot/integrations/discord.ts:94]: shouldn't this be

```suggestion
    if: () => self.guild.length > 0,
```

Comment (sogehige) [reply src/bot/integrations/discord.ts]: Also you can probably add alertbox for if: () => self.guild.length === 0 to inform that there are more settings after guild is set?

**Comment by Banonevar** [comment]
I cannot contitnue this PR because of https://github.com/sogehige/sogeBot/issues/3759 bug.

File: locales/en/ui/integrations/discord.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
     "joinToServerBtn": "Click to join bot to your server",
     "cannotJoinToServerBtn": "Set token and clientId to be able to join bot to your server",
     "noChannelSelected": "no channel selected",
-    "noRoleSelected": "no role selected"
+    "noRoleSelected": "no role selected",
+    "noGuildSelected": "no role selected",
```
Comment (sogehige) [inline locales/en/ui/integrations/discord.json:14]:
```suggestion
    "noGuildSelected": "no guild selected",
```

**Pull request merged**